tests/integration/test_static_analysis_warm_start_delta.py

```
@@ -0,0 +1,298 @@
+from pathlib import Path
+import subprocess
+
+import pytest
+
+import static_analyzer as static_analyzer_module
+from static_analyzer import StaticAnalyzer
+from static_analyzer.analysis_result import StaticAnalysisResults
+
+
+def _git(args: list[str], cwd: Path | None = None) -> str:
+    result = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)